src/sql/run.py

```diff
@@ -233,6 +233,13 @@ def pie(self, key_word_sep=" ", title=None, **kwargs):
         Any additional keyword arguments will be passed
         through to ``matplotlib.pylab.pie``.
         """
+        warnings.warn(
+            (
+                ".pie() will be removed in a future version,"
+                "please use %sqlplot pie. If you need help migrating,"
+                "send us a message: https://ploomber.io/community"
+            )
```

Comment:
Usually, if we are using the same text multiple times it's better to create a template and re-use it but since we'll deprecate and delete it, so we can leave it as is.

Reply:
Sure, got it!